tools/converter_coco_stuff.py

```diff
@@ -17,184 +17,23 @@
 
 from corenet.utils import logger
 
+# Generates remaps for COCO-Stuff labels to 171 classes.
+# Skips some of the labels that are not annotated.
+def generate_cocostuff_remap():
+    cocostuff_skips = {11, 25, 28, 29, 44, 65, 67, 70, 82, 90, 179, 181}  # IDs that are skipped
```

Comment:
Thanks for the PR, looks like `cocostuff_skips` is not completely correct. 179 and 181 are not skipped. By counting there are 11 categories skipped but `cocostuff_skips` have 12 elements.
```suggestion
    cocostuff_skips = {11, 25, 28, 29, 44, 65, 67, 68, 70, 82, 90}  # IDs that are skipped
```

Reply:
Thanks for the reply,

I must have overlooked that, sorry.